zigbee/Zigbee-Preparatory-Course-CN.md

```diff
@@ -0,0 +1,251 @@
+</div>  
+</br>  
+
+选择目标目录，单击[下一步]>，然后单击[安装]。
+
+应用程序启动时，将邀请您登录。请使用上面创建的支持帐户（或培训活动组织者提供的临时帐户）登录。尽管您可以在此处跳过登录，但是必须登录才能访问某些受保护的内容，例如EmberZNet PRO SDK。
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_login.png">  
+</div>  
+</br>  
+
+**注意**：如果您还没有任何WSTK，请忽略下面的两段。
+登录后，Simplicity Studio将检查连接的硬件。如果您通过USB电缆连接了WSTK，Simplicity Studio将检测USB电缆并提示您下载设备检查器。点击[是]。
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_install_required.png">  
+</div>  
+</br>  
+
+如果已连接WSTK，则将启动Launcher视图，如下所示。连接的设备将被标识为J-Link设备，并在“调试适配器”选项卡中列出，如图所示。
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_launcher.png">  
+</div>  
+</br>  
+
+### EmberZNet PRO堆栈
+Zigbee解决方案需要实现Zigbee无线和相关的微处理器（一起在单个芯片中或分开地），以及在Zigbee堆栈之上实现应用程序。EmberZNet PRO是Silicon Labs对Zigbee PRO协议栈的实现。借助Zigbee PRO堆栈，Silicon Labs既提供了基于Zigbee规范的标准网络API，同时也提供了应用程序框架，从而为客户提供了一种基于Zigbee应用程序配置文件和Zigbee群集库（ZCL）来快速开发其应用程序的方法。
+为了加快客户产品的上市速度，Silicon Labs提供了许多内置的演示和示例，涵盖了EmberZNet SDK中最常见的用例。
+
+假设您已经购买了网状网络开发套件，并且还使用您自己的帐户注册了序列号。使用相同的帐户登录Simplicity Studio，然后即可通过Simplicity Studio下载EmberZNet PRO堆栈。
+**注意**：如果您还没有开发工具包，也没有注册自己的帐户，请使用培训活动组织者提供的临时帐户登录Simplicity Studio。
+
+如果您尚未安装EmberZNet SDK，请运行以下程序包管理器以安装最新的EmberZNet SDK。
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_install_EmberZNet_SDK.gif">  
+</div>  
+</br>
+
+成功安装SDK后，单击[是]以重新启动Simplicity Studio，以使更改生效。
+而且，如果您安装了旧版本的EmberZNet SDK，则强烈建议将其更新为最新版本。其方法与上述类似。
+
+如果已连接WSTK，请在launcher左侧的[Debug Adapters]选项卡中单击设备，然后Launcher视图将展示您的硬件和与堆栈相关的软件组件和功能。
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_launcher_withEmberZnet.png">  
+</div>  
+</br>
+
+**注意**：如果还没有WSTK，则只需在Launcher视图左下角的“我的产品”选项卡中键入WSTK名称“ BRD4162A”。如果列出了多个版本条目，则选择“ Rev A01”。
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_launcher_no_hw.png">  
+</div>  
+</br>  
+
+* Demos  
+Demos是预构建的应用程序示例，可以直接下载到您的设备。  
+
+* Software Examples  
+软件示例提供了协议栈所需的默认配置以及基本应用程序框架。Silicon Labs建议您以基于用例的示例开始自己的开发，并根据需要对其进行修改。
+如果没有一个基于用例的示例满足您的需求，则可以从**ZigbeeMinimal**示例开始。我们还将使用**ZigbeeMinimal**作为Zigbee新手训练营系列动手学习的起点。
+
+* SDK Documentation  
+SDK文档列出了有关堆栈的文档，其中包括SDK版本说明，API参考等。
+
+### 编译器工具链：
+#### GCC
+Simplicity Studio提供了GCC（GNU编译器集合），它将在首次安装EmberZNet SDK时自动安装。
+您可以通过从Launcher视图单击[帮助]> [关于Simplicity Studio]> [工具链和SDK]，检查已安装的GCC工具链（以及SDK）的版本信息。
+
+<div align="center">
+  <img src="files/ZB-Zigbee-Preparatory-Course/preparatory_ss_sdks_toolchain.png">  
+</div>  
+
+#### IAR (非必需)
+建议在新手训练营系列培训中使用GCC，当然您也可以使用IAR来编译项目。请参阅EmberZNet SDK发行说明，了解兼容的IAR版本。
+对于Silicon Labs客户，您可以获得30天的IAR评估许可证：
+•转到Silicon Labs支持门户，网址为https://www.silabs.com/support。
+•向下滚动到页面底部，然后单击“联系支持”。
+•如果尚未登录，请登录。
+•单击“软件版本”选项卡。在视图列表中，选择开发工具。单击执行。结果是指向发行说明中命名的IAR-EWARM版本的链接。
+•下载IAR软件包（大约需要1个小时）。
+•安装IAR。
+•在“ IAR许可证向导”中，单击“向IAR Systems注册”以获取评估许可证。
+•完成注册，IAR将提供30天的评估许可证。
+•一旦安装了IAR-EWARM，下次Simplicity Studio启动时，它将自动检测并配置IDE以使用IAREWARM。
+
+## Github 帐户  
```

Comment:
![image](https://user-images.githubusercontent.com/1233397/77741902-97851f80-7050-11ea-98da-365c0feaf2cf.png)
change both en and cn to "GitHub"